internal/models/README-bullet-style.md

```
@@ -0,0 +1,124 @@
+
+## Overview
+The Bullet Style model represents different projectile types used in ammunition within The Virtual Armory application. Examples include FMJ (Full Metal Jacket), JHP (Jacketed Hollow Point), and SP (Soft Point).
+
+## Model Definition
+
+The BulletStyle model is defined in `internal/models/bullet_style.go`:
+
+```go
+type BulletStyle struct {
+	gorm.Model
+	Type       string `gorm:"type:varchar(100);not null;uniqueIndex" json:"type"`
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)